src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,306 @@
+same context. For example, if a website (`https://a.example`) opens a popup
+window (`https://b.example`), the opener window and the popup window share the
+same browsing context and they have access to each other via DOM APIs such as
+`window.opener`.
+
+![Browsing Context Group](browsing-context-group.png)
+
+As of Chrome 83, DevTools support is not yet available for COOP. However, you
+can examine `window.opener === null` from the opened window, or
+`openedWindow.closed === true` from the opener window to determine if they are
+in separate browsing context groups.
+
+{% Aside 'codelab' %}
+[See the impact of different COOP 
+params](https://first-party-test.glitch.me/coop).
+{% endAside %}
+
+#### 2. Ensure resources have CORP or CORS enabled
+
+Make sure that all resources in the page are loaded with CORP or CORS
+HTTP headers. This step 
+is required for [step four, enabling COEP](#enable-coep).
+
+Here is what you need to do depending on the nature of the resource:
+
+* If the resource is expected to be loaded **only from the same origin**, set 
+  the `Cross-Origin-Resource-Policy: same-origin` header.
+* If the resource is expected to be loaded **only from the same site but cross 
+  origin**, set the `Cross-Origin-Resource-Policy: same-site` header.
+* If the resource is **loaded from cross origin(s) under your control**, set the 
+  `Cross-Origin-Resource-Policy: cross-origin` header if possible. 
+* For cross origin resources that you have no control over:
+    * Use the `crossorigin` attribute in the loading HTML tag if the resource is
+      served with CORS.
+    * Ask the owner of the resource to support either CORS or CORP.
+* For iframes, use CORP and COEP headers as follows: 
+  `Cross-Origin-Resource-Policy: same-origin` and `Cross-Origin-Embedder-Policy: 
+  require-corp`.
+
+{% Aside 'key-term' %}
+It's important that you understand the difference between "same-site" and 
+"same-origin". Learn about the difference at [Understanding same-site and 
+same-origin](/same-site-same-origin).
+{% endAside %}
+
+#### 3. Use the COEP Report-Only HTTP header to assess embedded resources
+
+Before fully enabling COEP, you can do a dry run by using the
+`Cross-Origin-Embedder-Policy-Report-Only` header to examine whether the policy
+actually works. You will receive reports without blocking embedded content.
+Recursively apply this to all documents. For information on the Report-Only HTTP
+header, see [Observe issues using the Reporting
+API](#observe-issues-using-the-reporting-api).
+
+#### 4. Enable COEP {: #enable-coep }
+
+Once you've confirmed that everything works, and that all resources can be 
+successfully loaded, apply the `Cross-Origin-Embedder-Policy: require-corp` HTTP 
+header to all documents including those that are embedded via iframes.
+
+{% Aside 'codelab' %}
+[See the impact of different COEP / CORP 
+parameters](https://first-party-test.glitch.me/coep).
```

Comment:
I can set correct states and get a feedback that they are okay, but there is no feedback that something is set wrong especially for the cross origin iframe. I would have expected to see a similar result to the image: It is blocked and not loaded and it shows how Chrome would show it.

Reply:
I will sort it.

Reply:
I made some changes so it should look easier to understand.

Reply:
Thanks. This made it much clearer to me!